src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -52,6 +52,7 @@ internal class DebuggedProcess : MICore.Debugger
         private IProcessSequence _childProcessHandler;
         private bool _deleteEntryPointBreakpoint;
         private string _entryPointBreakpoint = string.Empty;
+        private bool _simpleValuesExcludesRefTypes = false;
```

Comment:
I would suggest making this a `bool?` and initializing it on first use rather than initializing it immediately on start debugging -- I am not sure if GDB will always be able to give you an accurate answer immediately on startup even before connecting.